telemetry/statsd/src/test/scala/io/buoyant/telemetry/statsd/MetricTest.scala

```diff
@@ -0,0 +1,98 @@
+package io.buoyant.telemetry.statsd
+
+import com.timgroup.statsd.StatsDClient
+import org.scalatest._
+
+class MetricTest extends FunSuite {
+
+  class MockStatsDClient extends StatsDClient {
```

Comment:
Consider moving this to its own file because its big and boilerplatey and it breaks up the flow.